### DISCUSSION TITLE
How to switch between emulated and native text tracks?

### DISCUSSION THREAD

**Discussion by OutofSkills** [videojs/video.js#8548] [Q&A]
I'm using the same player instance to play multiple types of videos (360, series) and in a certain case I want to only use emulated text tracks while for all others the native tracks. Is it possible?

Currently, I see that it works only when creating a new player instance and passing the options:
`
{
 html5: {
    nativeTextTracks: true
  }
}
`

**Comment by gkatsev** [comment]
I don't think there's a good way of doing it after init. I think it's mostly when the player is created, right now.

**Comment by OutofSkills** [comment]
Hello @gkatsev! Thanks for the answer!

While exploring the documentation, I found the [emulateTextTracks ](https://docs.videojs.com/tech#emulateTextTracks) method which seems to add the text tracks to DOM and with some css manipulation(as it seems that they are still hidden) it may do the trick.

Is it safe to use that method as a workaround?
